### ISSUE TITLE
failure with eatmydata "tried to call a logger before setup_logging() was called"

### ISSUE THREAD

**Issue by LocutusOfBorg** [borgbackup/borg#1111]
As said on Debian bug 825729, with eatmydata borg fails.
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=825729
```
==================================== ERRORS ====================================
___ ERROR collecting .pybuild/pythonX.Y_3.5/build/borg/testsuite/__init__.py ___
borg/testsuite/__init__.py:10: in <module>
    from ..xattr import get_all
borg/xattr.py:71: in <module>
    logger.error(msg)
borg/logger.py:173: in error
    return self.__logger.error(*args, **kw)
borg/logger.py:150: in __logger
    raise Exception("tried to call a logger before setup_logging() was called")
E   Exception: tried to call a logger before setup_logging() was called
--------------------------------------------------------------------------------
```

**Comment by ThomasWaldmann** [comment]
I've  no idea what eatmydata is, but it seems to invoke borg in an unusual way.

**Comment by LocutusOfBorg** [comment]
package description:
```
Library and utilities designed to disable fsync and friends
This package contains a small LD_PRELOAD library (libeatmydata) and a couple of helper utilities designed to transparently disable fsync and friends (like open(O_SYNC)). This has two side-effects: making software that writes data safely to disk a lot quicker and making this software no longer crash safe.
You will find eatmydata useful if particular software calls fsync(), sync() etc. frequently but the data it stores is not that valuable to you and you may afford losing it in case of system crash. Data-to-disk synchronization calls are typically very slow on modern file systems and their extensive usage might slow down software significantly. It does not make sense to accept such a hit in performance if data being manipulated is not very important.
On the other hand, do not use eatmydata when you care about what software stores or it manipulates important components of your system. The library is called libEAT-MY-DATA for a reason. 
```
Not sure if fixable or worth a fix :)

**Comment by enkore** [comment]
> Not sure if fixable or worth a fix :)

Not worth a fix. fsync()iness is fixed in 1.1.

Close?
